src/modules/addresses/address/access/AddressWorkerpoolsAccessTable.tsx

```diff
@@ -0,0 +1,128 @@
+import { PREVIEW_TABLE_LENGTH, TABLE_REFETCH_INTERVAL } from '@/config';
+import { useQuery } from '@tanstack/react-query';
+import { LoaderCircle } from 'lucide-react';
+import { DataTable } from '@/components/DataTable';
+import { PaginatedNavigation } from '@/components/PaginatedNavigation';
+import { getIExec } from '@/externals/iexecSdkClient';
+import { usePageParam } from '@/hooks/usePageParam';
+import { ErrorAlert } from '@/modules/ErrorAlert';
+import useUserStore from '@/stores/useUser.store';
+import { createPlaceholderDataFnForQueryKey } from '@/utils/createPlaceholderDataFnForQueryKey';
+import { columns } from './addressWorkerpoolColumns';
+
+function useAddressWorkerpoolsAccessData({
+  addressAddress,
+  currentPage,
+}: {
+  addressAddress: string;
+  currentPage: number;
+}) {
+  const { chainId } = useUserStore();
+
+  const pageSize = PREVIEW_TABLE_LENGTH * 2;
+
+  // API returns min 10 items, but we display only 5 per page
+  const apiBatch = Math.floor((currentPage * PREVIEW_TABLE_LENGTH) / pageSize);
+  const startIndexInBatch = (currentPage * PREVIEW_TABLE_LENGTH) % pageSize;
+
+  const queryKey = [
+    chainId,
+    'address',
+    'workerpoolsAccess',
+    addressAddress,
+    apiBatch,
+  ];
+
+  const { data, isLoading, isRefetching, isError, errorUpdateCount } = useQuery(
+    {
+      queryKey,
+      queryFn: async () => {
+        const iexec = await getIExec();
+
+        const { count, orders } =
+          await iexec.orderbook.fetchWorkerpoolOrderbook({
+            workerpool: 'any',
+            requester: addressAddress,
+            page: apiBatch,
+            pageSize,
+          });
```

Comment:
not sure this is functionnaly what we want
- this display access granted TO the user and not access granted BY the user
- some result may be filtered out by restrict not set to "any"